main.js

```diff
@@ -0,0 +1,41 @@
+var $car = document.querySelector('#car');
+var turnList = [
+  {
+    name: 'turn0',
+    i: 0
+  },
+  {
+    name: 'turn90',
+    i: 1
+  },
+  {
+    name: 'turn180',
+    i: 2
+  },
+  {
+    name: 'turn270',
+    i: 3
+  }
```

Comment:
I'm not sure what the point of the `i` property is here, you should probably just store the strings in the array directly, and you can use their array indexes for whatever you're using the `i` property for here.